k9mail/src/main/java/com/fsck/k9/ui/settings/general/GeneralSettingsDataStore.kt

```diff
@@ -41,7 +41,7 @@ class GeneralSettingsDataStore(
             "messageview_return_to_list" -> K9.messageViewReturnToList()
             "messageview_show_next" -> K9.messageViewShowNext()
             "quiet_time_enabled" -> K9.getQuietTimeEnabled()
-            "disable_notifications_during_quiet_time" -> K9.isNotificationDuringQuietTimeEnabled()
```

Comment:
This is a bug I introduced. Instead of renaming the variable/method in `K9` change the logic here to read
```kotlin
"disable_notifications_during_quiet_time" -> !K9.isNotificationDuringQuietTimeEnabled()
```